tex/gregoriotex-main.tex

```diff
@@ -1654,17 +1654,22 @@
 }%
 
 \def\gre@setgregoriofontscaled[#1]#2#3{%
-  \if\relax\detokenize{#1}\relax %
-    \xdef\gre@gregoriofontname{#2}%
-    \expandafter\xdef\csname gre@fontfactor@#2\endcsname{#3}%
-    \gre@loadgregoriofont%
-    \gdef\GreCPVirgaReversaAscendensOnDLine##1{##1}%
-  \else %
-    \xdef\gre@gregoriofontname{#2-#1}%
-    \expandafter\xdef\csname gre@fontfactor@#2-#1\endcsname{#3}%
-    \gre@loadgregoriofont%
-    \gdef\GreCPVirgaReversaAscendensOnDLine##1{\GreCPVirgaReversaLongqueueAscendens}%
-  \fi %
+  \IfStrEq{#2}{parmesan}{%%% DEPRECATED for removal in 5.0
+    \gre@deprecated{parmesan font}{granapadano font}%%% DEPRECATED for removal in 5.0
```

Comment:
I might add quotes around `parmesan` and `granapadano` in order to make it clear to the user what the appropriate substitution is.